cognite/client/data_classes/simulators/filters.py

```diff
@@ -59,3 +59,19 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         dumped = super().dump(camel_case=camel_case)
         dumped["property"] = self.property
         return dumped
+
+
+class SimulatorRoutineRevisionsFilter(CogniteFilter):
+    def __init__(
+        self,
+        routine_external_ids: Sequence[str] | None = None,
```

Comment:
Str is a sequence, what we have done other places in the SDK is using the `SequenceNotStr` from `from cognite.client.utils.useful_types import SequenceNotStr`
```suggestion
        routine_external_ids: SequenceNotStr[str] | None = None,
```